docs/user_guide/building_blocks/config_objects.md

```diff
@@ -1,31 +1,99 @@
+```python
+cfg = analysis.add_config(campaign, name=your_config_name, id=your_config_id)
+```
+to create the
+new Config object `cfg`, which will be
+associated to both the Analysis object and the Campaign object needed for its creation. As the
+Config object should contain the analysis-dependent information related to a certain campaign, it
+should contain most of the information needed for running your analysis. Therefore, in this section,
+the Config parameters required by Columnflow and some convenience parameters will be presented.
+
+To start your analysis, do not forget to use the already existing analysis template in the
+`analysis_templates/cms_minimal` Git directory and its
+[config](https://github.com/columnflow/columnflow/blob/master/analysis_templates/cms_minimal/__cf_module_name__/config/analysis___cf_short_name_lc__.py).
+
+
+The Config saves informations under two general formats: the objects
+from the order package, which are necessary for your analysis to run in columnflow, and the
+additional parameters, which are saved under the auxiliary key, accessible through the "x" key.
+Unlike the general order package objects and while several names for parameters in the auxiliary
+field do already have a meaning in columnflow and should respect the format used in columnflow,
+the auxiliary field can contain any kind of
+parameter the user wants to save and reuse for parts of the analysis. These two general formats are
+presented below.
+
+It is generally advised to create functions to separate the different parts of the filling of the Config object, especially as some parts will not change very often once the analysis is set up (e.g. luminosity), while others will be changed quite often, e.g. each time new variables are created. An example of such a separation can be found in the existing [hh2bbtautau analysis](https://github.com/uhh-cms/hh2bbtautau).
 
-## Processes
 
-TODO
+## Parameters from the order package (required)
+
+### Processes
+
+The physical processes to be investigated in the analysis. These should be saved as objects of the
+{external+order:py:class}`order.process.Process` class and added to the Config object using its
+{external+order:py:meth}`order.config.Config.add_process()` method. The processes added to the
+Config must correspond to the
+processes defined for the datasets and added to the Campaign object associated to the Config.
+It is possible to get all root processes from a specific campaign using the
+{py:func}`~columnflow.config_util.get_root_processes_from_campaign()` function from columnflow.
+As an example of information carried by a process, a color for the plotting scripts can be set using
+the {external+order:py:attr}`order.mixins.ColorMixin.color1` attribute of the process.
+More informations about processes can be found in the
+{external+order:py:class}`order.process.Process` and the
+{external+order:doc}`quickstart` sections of the order documentation.
```

Comment:
It might be nice to mention that processes also carry the cross section information and that we can use that to determine normalization weights, linking to our normalization_weights Producer as an example

Reply:
Also, it might be nice to link to some example in the cmsdb to showcase how the implementation of a process might look like (same for datasets)

Reply:
examples from the order docu are linked, I add a link to the cmsdb example provided in structure.md. Cross section information has been added